src/v0/destinations/active_campaign/util.js

```diff
@@ -1,19 +1,24 @@
 const {
   nodeSysErrorToStatus
 } = require("../../../adapters/utils/networkUtils");
-const { CustomError } = require("../../util");
+const {
+  RetryableError,
+  NetworkInstrumentationError
+} = require("../../util/errorTypes");
 
 const errorHandler = (err, message) => {
   if (err.response) {
-    throw new CustomError(
+    throw new NetworkInstrumentationError(
```

Comment:
How do we know instrumentation is the problem?

Reply:
Here, if we are getting a false success Response for an axios call. This only seems the best suite.

Reply:
Instrumentation error puts blame on the user. Unless we know for sure, we shouldn't do that.
You can use `AbortedError` and pass the status code to it.

Reply:
Actually, use `NetworkError` and deduce the error type via `getDynamicErrorType` method. See the parent PR for reference.